lib/internal/bootstrap/loaders.js

```diff
@@ -164,6 +167,7 @@ NativeModule.exposeInternals = function() {
     // Do not expose this to user land even with --expose-internals.
     if (id !== loaderId) {
       mod.canBeRequiredByUsers = true;
+      mod.defaultOnly = true;
```

Comment:
Why do we need to special case this for `loaders.js`?

Reply:
Ok I've inlined this check as you suggest in the next comment.